src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, number = 0) {
+      callback(number);
```

Comment:
do not forget to bind `this` to callback

Reply:
Thanks. Done.